dodo.py

```diff
@@ -138,6 +138,8 @@ def task_install_release():
     def install_release(asversion, force):
         platform = sys.platform
         if platform.startswith('linux'):
+            removeDevDirCache=Path(os.environ["HOME"])/".cache"/"hxadl"/"dev"
```

Comment:
These lines removing the installed directory/dev are not necessary - instead see option "--force" on the `install_release`.